install_config/master_node_configuration.adoc

```diff
@@ -1761,39 +1763,63 @@ $ openshift start node --config=/openshift.local.config/node-<node_hostname>/nod
 |8|Body-level API debugging information
 |===
 
-You can control which INFO messages are logged by setting the loglevel option in the in xref:../admin_guide/manage_nodes.adoc#modifying-nodes[node configuration files], 
-the *_/etc/origin/master/master-config.yaml_* file. Configuring the logs to collect all messages can lead to large logs that are difficult to interpret and can take up excessive space. Collecting all messages should only be used in debug situations.
+You can xref:master-node-config-logging-levels[change the log levels independently for masters or nodes] as needed.
 
-[NOTE]
-====
-Messages with FATAL, ERROR, WARNING and some INFO severities appear in the logs regardless of the log configuration.
-====
+*View node logs*
 
-You can view logs for the master or the node system using the following command:
+To view logs for the node system, run the following command:
 
 ----
 # journalctl -r -u <journal_name>
 ----
 
 Use the `-r` option to show the newest entries first.
 
+*View master logs*
+
+To view logs for the master components, run the following command:
+
+----
+# /usr/local/bin/master-logs <component> <controller> <1>
+----
+
+<1> Specify the controller and the name of the container. Valid component values are `controllers`, \api\, and `etcd`. 
```

Comment:
"Specify the controller" is not correct.
`\api\` has syntax issue.
IMO, it is OK to just remove Line 1786 and the above <1>, because below "For example" listed all examples